### PR TITLE
EO-428: Create Health Score Container Component (Overview)

### PR DESCRIPTION
**Pull request by bkemper**

### CODE REVIEW

File: src/actions/signals.js [hunk 1]
```diff
@@ -54,12 +54,17 @@ const signalsActionCreator = ({ dimension, type }) => ({
...
-  dimension: 'spam-hits',
-  type: 'GET_SPAM_HITS'
-});
-
 export const getEngagementRecency = signalsActionCreator({
   dimension: 'cohort-engagement',
   type: 'GET_ENGAGEMENT_RECENCY'
 });
+
+export const getHealthScore = signalsActionCreator({
```
Comment (jonambas) [inline src/actions/signals.js]: This mapping seems to work 
```
current_health_score: 'health_score'
```

File: src/selectors/signals.js [hunk 2]
```diff
@@ -174,6 +175,49 @@ export const selectEngagementRecencyOverview = createSelector(
...
+      },
+      now: moment().subtract(1, 'day'),
+      relativeRange
+    });
+
+    return {
+      ...rowOfData,
+      current_health_score: _.last(filledHistory).health_score,
+      history: filledHistory,
+      average_health_score: Math.floor(
```
Comment (jonambas) [inline src/selectors/signals.js]: Are we not doing change week over week?

Comment (bkemper) [reply src/selectors/signals.js]: Eventually, there are tickets (e.g. EO-477) to implement it on API first.

**Comment by bkemper** [comment]
Rebased and resolved merge conflicts.

**Comment by bkemper** [comment]
Rebased with master.

**Pull request merged**